lokikit/commands.py

```
@@ -641,3 +656,372 @@ def force_quit_command(ctx):
+
+def parse_command(ctx, directory: str, dashboard_name: str | None = None, max_files: int = 5, max_lines: int = 500) -> None:
+    """Parse logs and interactively create Grafana dashboards.
+
+    Args:
+        ctx: Click context
+        directory: Directory containing log files to parse
+        dashboard_name: Name for the generated dashboard
+        max_files: Maximum number of log files to sample
+        max_lines: Maximum number of lines to sample per file
+    """
+    import glob
```

Ruff (0.8.2)

Redefinition of unused `glob` from line 3

Remove definition: `glob`

(F811)